zad2/js/scripts.js

```diff
@@ -0,0 +1,13 @@
+$(function(){
+	var carouselList = $("#carousel ul");
+		interval = setInterval(changeSlide, 3000);
+	function changeSlide() {
+		carouselList.animate({'marginLeft':-1500}, 8000, moveFirstSlide);
```

Comment:
dlaczego `'marginLeft':-1500`?

Reply:
bo w sumie długość 6 slajdów u mnie to 1800 ale jak jest ten przeskok na pierwszy to przy dl 1800 jest pusty biały slajd więc 1500

Reply:
ale teraz przewijasz wszystkie slajdy naraz, a chcesz przewinac tylko jeden